samples/RefreshFlow/AngularApp/package.json

```diff
@@ -1,39 +1,44 @@
 {
-  "name": "openiddict-samples.refresh-flow.angularapp",
-  "version": "1.0.0",
+  "name": "angularapp",
+  "version": "0.0.0",
+  "license": "MIT",
   "angular-cli": {},
   "scripts": {
-    "start": "ng serve --port 5055",
-    "lint": "tslint \"src/**/*.ts\""
+    "ng": "ng",
+    "start": "ng serve --port 5055"
   },
+  "private": true,
   "dependencies": {
-    "@angular/common": "^2.2.0",
-    "@angular/compiler": "^2.2.0",
-    "@angular/core": "^2.2.0",
-    "@angular/forms": "^2.2.0",
-    "@angular/http": "^2.2.0",
-    "@angular/platform-browser": "^2.2.0",
-    "@angular/platform-browser-dynamic": "^2.2.0",
-    "@angular/router": "^3.2.0",
+    "@angular/common": "^2.3.1",
+    "@angular/compiler": "^2.3.1",
+    "@angular/core": "^2.3.1",
+    "@angular/forms": "^2.3.1",
+    "@angular/http": "^2.3.1",
+    "@angular/platform-browser": "^2.3.1",
+    "@angular/platform-browser-dynamic": "^2.3.1",
+    "@angular/router": "^3.3.1",
     "core-js": "^2.4.1",
-    "rxjs": "5.0.0-beta.12",
+    "rxjs": "^5.0.1",
     "ts-helpers": "^1.1.1",
-    "zone.js": "^0.6.23",
-    "@ngrx/core": "1.2.0",
-    "@ngrx/devtools": "~1.4.0",
-    "@ngrx/store": "2.2.1",
-    "@ngrx/store-devtools": "~3.2.1",
-    "angular2-jwt": "^0.1.24"
+    "zone.js": "^0.7.2",
+    "jwt-decode": "2.1.0"
   },
   "devDependencies": {
-    "@angular/compiler-cli": "^2.1.0",
+    "@angular/compiler-cli": "^2.3.1",
+    "@types/jasmine": "2.5.38",
```

Comment:
Consider deleting the parts of this package.json that do not add clear value. E.g. 

```
-  "angular-cli": {},
-  "private": true,
-    "@types/jasmine": "2.5.38",
-    "jasmine-core": "2.5.2",          
-    "jasmine-spec-reporter": "2.5.0", 
-    "karma": "1.2.0",                 
-    "karma-chrome-launcher": "^2.0.0",
-    "karma-cli": "^1.0.1",            
-    "karma-jasmine": "^1.0.2",        
-    "karma-remap-istanbul": "^0.2.1", 
-    "protractor": "~4.0.13",          
```

Deleting the non-essential stuff will make it easier to review the code.